ggml/src/ggml-vulkan/vulkan-shaders/vulkan-shaders-gen.cpp

```diff
@@ -250,7 +272,15 @@ void string_to_spv_func(const std::string& _name, const std::string& in_fname, c
         // }
         // std::cout << std::endl;
 
-        execute_command(command, stdout_str, stderr_str);
+        execute_command(
+            #ifdef _WIN32
+            command,
```

Comment:
Maybe move the loop at line 263 into execute_command and just pass the cmd vector here.